plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/ext/provider/ApiEndpointEnvVariableProvider.java

```diff
@@ -30,6 +32,11 @@
 
     @Override
     public String get() {
+        String apiEndpointEnvVar = System.getenv(DockerInstanceRuntimeInfo.API_ENDPOINT_URL_VARIABLE);
```

Comment:
@l0rd Can you describe why you want to add that functionality and how it will be used